doc-content/enterprise-only/openshift/registry-authentication-proc.adoc

```diff
@@ -0,0 +1,24 @@
+[id='registry-authentication-proc']
+= Ensuring your environment is authenticated to the Red Hat registry
```

Comment:
This section is optional. It's only necessary for clusters that don't already have the rhpam/rhdm 7.3 imagestreams available in the `openshift` project space.